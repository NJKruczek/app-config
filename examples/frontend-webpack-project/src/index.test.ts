/**
 * @jest-environment jsdom
 */

const logSpy = jest.spyOn(console, 'log');

const config = {
  externalApiUrl: 'http://localhost:3002',
};

jest.mock('@lcdev/app-config', () => ({ config, default: config }));

import './index';

it('logs config', () => {
  expect(logSpy).toHaveBeenCalledWith('externalApiUrl:', config.externalApiUrl);
  expect(document.body).toMatchSnapshot();
});
