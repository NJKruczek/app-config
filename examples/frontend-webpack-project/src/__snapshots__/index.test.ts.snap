// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`logs config 1`] = `
<body>
  {"externalApiUrl":"http://localhost:3002"}
</body>
`;
